src/components/forms/EventStreamsForm.js

```diff
@@ -0,0 +1,186 @@
+import React, { Component } from "react";
+import {
+  buildFormDefaultInputMethods,
+  buildFormFunctions,
+} from "../component-utils";
+import { IcseNameInput } from "../Inputs";
+import { EndpointSelect, IcseSelect } from "../Dropdowns";
+import { IcseFormGroup } from "../Utils";
+import { ToolTipWrapper } from "../Tooltips";
+import { isIpStringInvalid } from "../../lib/iam-utils";
+import { TextArea } from "@carbon/react";
+import { titleCase } from "lazy-z";
+import PropTypes from "prop-types";
+
+/**
+ * EventStreamsForm
+ */
+class EventStreamsForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = this.props.data;
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleAllowedIps = this.handleAllowedIps.bind(this);
+    this.handlePlanChange = this.handlePlanChange.bind(this);
+    buildFormFunctions(this);
+    buildFormDefaultInputMethods(this);
+  }
+
+  /**
+   * Handle input change for allowed ips text field
+   * @param {event} event
+   */
+  handleAllowedIps(event) {
+    // removing white space and checking for empty value
+    let value = event.target.value.replace(/\s*/g, "");
+    if (value === "") value = null;
```

Comment:
why set to `null` here? I think maybe makes more sense to set to `""`

Reply:
I believe we set to null here sot that the placeholder of `X.X.X.X, X.X.X.X/X, ...` shows when no value is present in the text area.
```js
placeholder={ this.state.private_ip_allowlist || "X.X.X.X, X.X.X.X/X, ..." }
```

Reply:
actually i think we can have `""`